src/http/mod.rs

```diff
@@ -2,6 +2,7 @@ pub(self) mod cf;
 pub(crate) mod feature;
 pub(self) mod legacy;
 
+pub const DEFAULT_HTTP_TIMEOUT_SECONDS: u64 = 5 * 60;
```

Comment:
This is 5 minutes, not 1 minute, right? Or am I misunderstanding how this value is used?

Reply:
bah. you are correct. the  five minutes should be for the  other thing.